doc/mgr/smb.rst

```diff
@@ -357,6 +357,21 @@ custom_dns
 placement
     Optional. A Ceph Orchestration :ref:`placement specifier
     <orchestrator-cli-placement-spec>`.  Defaults to one host if not provided
+custom_smb_share_options
+    Optional mapping. Specify key-value pairs that will be directly added to
```

Comment:
Sentence please

Reply:
Would something like 'Optional mapping, defaulting to empty.' be OK so as to match the other blocks in this definition list?

Reply:
(ping) I'd like an opinion from you on this because I want to be consistent with the existing options. `Optional mapping.` is more consistent despite being a sentence fragment.